packages/mobile/src/services/offline-downloader/offline-sync.ts

```diff
@@ -170,7 +169,11 @@ export const syncCollectionTracks = async (
     updatedCollection
   )
 
-  downloadCollection(updatedCollectionWithArt)
+  downloadCollection(
+    updatedCollectionWithArt,
```

Comment:
in future probably good to have these args be spread objects so it's less work to refactor